aws/scripts/py/deploy_content.py

```
@@ -2,24 +2,25 @@
-    return {
-        "production": os.environ['BUCKET_NAME'],
-        "staging": os.environ['STAGING_BUCKET_NAME'],
-    }
+def get_bucket():
+    print("Getting bucket...")
+    return os.environ["BUCKET_NAME"]
 
 
 def fetch_distributions():
     print("Fetching CloudFront distributions...")
     result = subprocess.check_output(
```

Ruff (0.8.2)

`subprocess` call: check for execution of untrusted input

(S603)